Services/Interfaces/web_interface/controllers/configuration.py

```diff
@@ -36,12 +36,19 @@
 def profile():
     selected_profile = flask.request.args.get("select", None)
     next_url = flask.request.args.get("next", None)
+    reboot = flask.request.args.get("reboot", False)
+    reboot_delay = 3
     if selected_profile is not None and selected_profile != models.get_current_profile().profile_id:
         models.select_profile(selected_profile)
         current_profile = models.get_current_profile()
-        flask.flash(f"Switched to {current_profile.name} profile.", "success")
+        if not reboot:
```

Comment:
Should we flash something before rebooting like "Applying configuration..." ?

Reply:
there is a waiter card that appears when rebooting